### PR TITLE
LPS-107504 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
Tests passed here: https://github.com/lawrence-lee/liferay-portal/pull/145#issuecomment-578277113

File: modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java [hunk 1]
```diff
@@ -3946,6 +4526,53 @@ else if (liferayVersion.equals("7.1.3")) {
 		}
 	}
 
+	private void _testBuildTemplateRestLatest(String liferayVersion)
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java]:
```suggestion
	private void _testBuildTemplateRest(String liferayVersion)
```

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/hhuijser/liferay-portal/pull/3988. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/hhuijser/liferay-portal/pull/3989. [See changes here.](https://github.com/gamerson/liferay-portal/compare/79fa706f1a40~2...79fa706f1a40) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**